parquet-column/src/main/java/org/apache/parquet/internal/column/columnindex/ColumnIndexBuilder.java

```diff
@@ -673,6 +667,6 @@ public int getPageCount() {
    * @return the sum of size in bytes of the min/max values added so far to this builder
    */
   public long getMinMaxSize() {
-    return minMaxSize;
+    return 0;
```

Comment:
Should we throw instead?

Reply:
Changed to throw `UnsupportedOperationException`